Scheduler/observing_blocks_maintel/BLOCK-T299.json

```diff
@@ -0,0 +1,48 @@
+{
+    "name": "BLOCK-T299",
+    "program": "BLOCK-T299",
+    "constraints": [],
+    "scripts": [
+          {
+            "name": "maintel/point_azel.py",
+            "standard": true,
+            "parameters": {
+                "az": 295.713,
```

Comment:
Since the az/el is so time sensitive, this is a great place to use a configurable block. See this example of BLOCK-T284 which uses a configurable azimuth https://github.com/lsst-ts/ts_config_ocs/blob/develop/Scheduler/observing_blocks_maintel/BLOCK-T284.json, and the notebook used to create it here https://github.com/lsst-ts/ts_block_utils/blob/develop/notebooks/maintel/BLOCK-T284.ipynb

Reply:
In this case, since the satellite is own geosynchronous orbit and in a fixed position of the sky , we don't need to modify the position.  I've used the mean of the reported position after collecting (what I think is the correct) data for a few weeks.  We can't predict where it will be in the future do this precision as they are actively moving the satellite around to keep it in position.

Here is the data from a few weeks (you can see it is very tightly constrained):

![image](https://github.com/user-attachments/assets/b388a1e6-c004-44b0-bd55-70300951c1a0)